Formula/opencv@3.rb

```diff
@@ -77,10 +78,12 @@ def install
 
     mkdir "build" do
       system "cmake", "..", *args
+      inreplace "modules/core/version_string.inc", "#{HOMEBREW_SHIMS_PATH}/mac/super/", ""
```

Comment:
Does this still use shims at build-time?

Reply:
Same ⬆️